icu4j/build.properties

```diff
@@ -7,6 +7,6 @@
 api.report.version = 74
 api.report.prev.version = 73
 release.file.ver = 74rc
```

Comment:
FYI: I am curious whether this value is still used.

Reply:
No, it is not, the whole file will be removed.